src/views/LoginView.js

```diff
@@ -1,34 +1,45 @@
 import React from 'react';
-import {Header, Icon, Image, Grid, Message, Segment} from 'semantic-ui-react';
-import {LoginCard} from '../components/LoginCard';
+import { Header, Icon, Image, Grid, Responsive, Segment } from 'semantic-ui-react';
+import { LoginCard } from '../components/LoginCard';
 import logo from '../assets/logo.svg';
 
+
 const LoginView = () => {
   return (
-    <Grid
-      stretched
-      className="View--Login"
-      textAlign="center"
-      style={{height: '100vh'}}
-      verticalAlign="middle"
-    >
-      <Grid.Column computer="8" tablet="12" mobile="15">
-        <Segment>
-          <Header as="h1">
-            <Image src={logo} />
-            <Header.Content>Kids First Data Tracker</Header.Content>
-          </Header>
-          <LoginCard />
-          <Message>
-            New to Kids First Data Tracker?
-            <a href="/">
-              Join now <Icon name="chevron right" />
-            </a>
-          </Message>
-        </Segment>
-      </Grid.Column>
-    </Grid>
+    <section className="View--Login">
+      <Grid
+        centered
+        columns={12}
+        className="h-screen"
+      >
+        <Grid.Row columns={12} verticalAlign="middle" >
+          <Grid.Column computer={3} tablet={8} mobile={14} textAlign="center">
+            <Header as="h1">
+              <Image src={logo} />
+              <Header.Content>Kids First Data Tracker</Header.Content>
+            </Header>
+
+            <Segment.Group raised >
+              <Segment>
+                <Header as="h3"> Log In</Header>
+                <LoginCard />
+              </Segment>
+              <Segment secondary>
+                <p>
+                  New to Kids First Data Tracker?
+                  &nbsp;
+                  <Responsive as="br" maxWidth={1770} />
```

Comment:
Could use the predefined breakpoints here: `Responsive.onlyLargeScreen.minWidth`